src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/repackage/org/apache/commons/httpclient/contrib/ssl/EasySSLProtocolSocketFactory.java

```diff
@@ -174,7 +174,7 @@ public Socket createSocket(Socket socket, String host, int port, boolean autoClo
   }
 
   public boolean equals(Object obj) {
-    return ((obj != null) && obj.getClass().equals(EasySSLProtocolSocketFactory.class));
```

Comment:
I wouldn't want to change the copied code in any way, that is the reason to have findbugs exclude in the first place. 
I would rather leave the exception for `~stashpullrequestbuilder\.stashpullrequestbuilder\.repackage\..*` or wait until #68 is merged.

Reply:
OK, removing that part. It's a small PR now, but at least I know that the remaining suppressions are still needed.

Reply:
Fixed